dense-passage-retrieval-with-ann/README.md

```diff
@@ -481,12 +475,13 @@ rank-profile openqa {
 }
 </pre>
 
-The *input_ids* function builds the input tensor to the ONNX model. The batch
-size is 1 and the max sequence length is 380 token _ids, including special
-tokens like CLS and SEP. The function builds a tensor: [[CLS, question
-token_ids, SEP, title_token_ids, SEP, text_token_ids]] which is evaluated by
-the Reader ONNX model. The **summary-features** is a way to pass ranking
-features and tensors from the content nodes to the java serving container.
+The *input_ids* function builds the input tensor to the ONNX model.
+The batch size is 1 and the max sequence length is 380 token _ids,
+including special tokens like CLS and SEP.
+The function builds a tensor: [[CLS, question_token_ids, SEP, title_token_ids, SEP, text_token_ids]]
```

Comment:
Assuming you meant _question_token_ids_ here

Reply:
Yes, looks correct.